scanrepository/scanrepository.go

```diff
@@ -456,6 +463,36 @@ func (cfp *ScanRepositoryCmd) openAggregatedPullRequest(repository *utils.Reposi
 	return cfp.handleFixPullRequestContent(repository, fixBranchName, pullRequestInfo, vulnerabilities...)
 }
 
+func (cfp *ScanRepositoryCmd) cleanNewFilesMissingInRemote() error {
```

Comment:
Can we add a UT for this function?
looks like multiple things can go wrong - so ideally I'd like to see a test case for the 3 errors + 1 happy flow - with some files that stayed and some that deleted